Tasks/InstallAppleProvisioningProfileV1/installprovprofile.ts

```diff
@@ -23,15 +23,20 @@ async function run() {
 
             if (tl.filePathSupplied('provProfileSourceRepository') && tl.exist(provProfilePath) && tl.stats(provProfilePath).isFile()) {
                 const info = await sign.installProvisioningProfile(provProfilePath);
-                tl.setTaskVariable('APPLE_PROV_PROFILE_UUID', info.provProfileUUID);
+                if (info && info.provProfileUUID) {
+                    tl.setTaskVariable('APPLE_PROV_PROFILE_UUID', info.provProfileUUID.trim());
```

Comment:
Why not just have `installProvisioningProfile` from ios-signing-common return a trimmed string?